docs/source/conf.py

```diff
@@ -33,6 +54,41 @@
 
 napoleon_custom_sections = ["Features"]
 
+parent_path = os.path.abspath(os.path.join(ROOT_DIR, os.pardir))
+
+sphinx_gallery_conf = {
+    "examples_dirs": f"{parent_path}/examples/desktop",
+    "gallery_dirs": "_gallery",
+    "backreferences_dir": "_gallery/backreferences",
+    # "exclude_implicit_doc": {
+    #     "WgpuRenderer",
+    #     "Resource",
+    #     "WorldObject",
+    #     "Geometry",
+    #     "Material",
+    #     "Controller",
+    #     "Camera",
+    #     "show",
+    #     "Display",
+    #     "Group",
+    #     "Scene",
+    #     "Light",
+    # },
+    "doc_module": ("fastplotlib",),
+    "image_scrapers": ("fastplotlib",),
```

Comment:
If you put "pygfx" here instead, I *think* you can use our scraper, so you don;t need the `gallery_scraper.py`.